src/EmployerAan/SFA.DAS.EmployerAan/Configuration/AanHubApiConfiguration.cs

```diff
@@ -1,10 +1,3 @@
-﻿using System.Diagnostics.CodeAnalysis;
+﻿namespace SFA.DAS.EmployerAan.Configuration;
 
-namespace SFA.DAS.EmployerAan.Configuration;
-
-[ExcludeFromCodeCoverage]
-public class AanHubApiConfiguration
-{
-    public string Url { get; set; } = null!;
-    public string Identifier { get; set; } = null!;
-}
\ No newline at end of file
+public class AanHubApiConfiguration : InnerApiConfiguration { }
```

Comment:
Remove the unnecessary use of base class from shared project. This base class is written for shared clients which we are not using.